data/json/items/armor/coats.json

```diff
@@ -1399,6 +1399,35 @@
     "flags": [ "VARSIZE" ],
     "armor": [ { "encumbrance": 10, "coverage": 80, "covers": [ "leg_l", "leg_r", "torso", "arm_l", "arm_r" ] } ]
   },
+  {
+    "id": "leather_police_jacket",
+    "type": "ARMOR",
+    "name": { "str": "leather police jacket" },
+    "description": "A thick and heavy leather jacket emblazoned with the insignia of the local police force.",
+	"repairs_like": "jacket_leather",
+    "weight": "1450 g",
+    "volume": "4500 ml",
+    "price": 17900,
+    "price_postapoc": 500,
+    "to_hit": 1,
```

Comment:
Why would a jacked have a bonus to hit someone with it?

Reply:
Beats me. I merely based its stats on the regular leather jacket.